src/Resources/views/CRUD/Association/edit_one_to_many_inline_table.html.twig

```diff
@@ -23,9 +23,12 @@ file that was distributed with this source code.
                             style="display:none;"
                         {% endif %}
                     >
-                        {{ nested_field.vars.label|trans({}, nested_field.vars['sonata_admin'].admin.translationDomain
-                            |default(nested_field.vars.translation_domain)
-                        ) }}
+                        {% if nested_field.vars.translation_domain is same as(false)  %}
+                            {{ nested_field.vars.label }}
+                        {% else %}
+                            {% set translationDomain = nested_field.vars.translation_domain|default(nested_field.vars['sonata_admin'].admin.translationDomain) %}
```

Comment:
I'd prefer to avoid one time variables, but I guess this is better for readability.

Reply:
This seems so much more readable this way.

Reply:
Yeah I prefer to avoid them too, but I agree it's more readable. Tell me if you want to change ;)